forge-game/src/main/java/forge/game/Game.java

```diff
@@ -712,6 +712,9 @@ public void forEachCardInGame(Visitor<Card> visitor, boolean withSideboard) {
             if (!visitor.visitAll(player.getInboundTokens())) {
                 return;
             }
+            if (!visitor.visitAll(player.getZone(ZoneType.None).getCards())) {
```

Comment:
this might cause all sorts of problems, let's not make None zone more real than it needs to be